### PR TITLE
Feat/mapr610 di

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [X] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [X] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [X] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**
MapR 6.1.0 implementation for DI

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.mapr610/src/org/talend/hadoop/distribution/mapr610/MapR610Distribution.java [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+
+    @Override
+    public boolean doSupportOldImportMode() {
+        return false;
+    };
+
+    @Override
+    public Set<ESparkVersion> getSparkVersions() {
+        Set<ESparkVersion> version = new HashSet<>();
+        version.add(ESparkVersion.SPARK_2_2);
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.mapr610/src/org/talend/hadoop/distribution/mapr610/MapR610Distribution.java]: How about Spark 2.3? I see Spark 2.3.2.0 in https://mapr.com/docs/61/InteropMatrix/Component_versions_all_MEPs.html for MEP 6.1

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml [hunk 2]
```diff
@@ -0,0 +1,1457 @@
...
+            mvn_uri="mvn:org.talend.libraries/winutils-hadoop-2.7.0/6.0.0"/>
+        <libraryNeeded
+            context="plugin:org.talend.hadoop.distribution.mapr.6.1.0"
+            id="mapr-streams-6.1.0-mapr.jar"
+            name="mapr-streams-6.1.0-mapr.jar"
+            mvn_uri="mvn:com.mapr.streams/mapr-streams/6.1.0-mapr"/>
+        <libraryNeeded
+            context="plugin:org.talend.libraries.hadoop.mapr.6.1.0"
+            id="datanucleus-api-jdo-4.2.4.jar"
+            name="datanucleus-api-jdo-4.2.4.jar" mvn_uri="mvn:org.datanucleus/datanucleus-api-jdo/4.2.4" >
```
[the remark this answers is not in the document]
Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: Actually, this format comes from 5.2 (do not know why). Anyway, fixed

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml [hunk 3]
```diff
@@ -0,0 +1,1457 @@
...
+            mvn_uri="mvn:org.apache.hive/hive-jdbc/2.3.3-mapr-1901" >
+       </libraryNeeded>
+       <libraryNeeded
+            context="plugin:org.talend.libraries.hadoop.mapr.6.1.0"
+            id="hive-service-2.3.3-mapr-1901.jar"
+            name="hive-service-2.3.3-mapr-1901.jar"
+            mvn_uri="mvn:org.apache.hive/hive-service/2.3.3-mapr-1901" >
+        </libraryNeeded>
+
+        <!--libraryNeeded
```
[the remark this answers is not in the document]
Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: Fixed

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml [hunk 4]
```diff
@@ -0,0 +1,1457 @@
...
+        	mvn_uri="mvn:org.apache.hadoop/hadoop-azure/2.7.0-mapr-1808"/>
+
+        <libraryNeeded context="plugin:org.talend.hadoop.distribution.mapr.6.1.0" 
+        	id="jetty-util-6.1.26.jar" 
+        	name="jetty-util-6.1.26.jar" 
+        	mvn_uri="mvn:org.mortbay.jetty/jetty-util/6.1.26"/>
+
+		<!--let try datalake on mapr 6.1.0 !!! -->
+        <libraryNeeded
+            context="plugin:org.talend.hadoop.distribution.cdh5120"
```
[the remark this answers is not in the document]
Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: Fixed

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: Thanks!

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: Thanks!

File: main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.mapr610/plugin.xml]: Thanks!

File: main/plugins/org.talend.hadoop.distribution.mapr610/src/org/talend/hadoop/distribution/mapr610/MapR610Distribution.java (hunk 1 above)
Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.mapr610/src/org/talend/hadoop/distribution/mapr610/MapR610Distribution.java]: Splendid, thanks!

**Pull request merged**